GDeflate/GDeflateDemo/main.cpp

```diff
@@ -59,7 +59,7 @@ enum class Operation
 struct Options
 {
     bool ShowHelp = false;
-    Operation Operation = Operation::None;
+    enum Operation Operation = Operation::None;
```

Comment:
Thanks for this.  TBH, I'd prefer to see the enum name changed (or the field changed, but I can think of a new name for the enum - say "OperationType" - more easily than a new name for the field) rather than use the enum qualifier here.

Reply:
Sure, I've updated the commit (force pushed to my fork).

Reply:
Thank you!